JShellAPI/src/test/java/org/togetherjava/jshellapi/JShellApiTests.java

```diff
@@ -0,0 +1,90 @@
+package org.togetherjava.jshellapi;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import org.togetherjava.jshellapi.dto.JShellResult;
+import org.togetherjava.jshellapi.dto.JShellSnippetResult;
+import org.togetherjava.jshellapi.dto.SnippetStatus;
+import org.togetherjava.jshellapi.dto.SnippetType;
+import org.togetherjava.jshellapi.rest.ApiEndpoints;
+
+import java.time.Duration;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * This class holds integration tests for JShellAPI. It depends on gradle building image task, fore
+ * more information check "test" section in gradle.build file.
```

Comment:
> This class holds integration tests for JShellAPI.

use "Integrates tests for JShellAPI." instead. Don't use the term class, it's redondant.
> fore

for*
>   check "test" section in gradle.build file.

Why do you need to specify this ?

Reply:
> Why do you need to specify this ?

Somebody may not notice the additional configs in _build.gradle_ file about tests

Reply:
How can you not notice it ? You need to check build.gradle to see how the tests work.